client_configuration.go

```diff
@@ -26,21 +27,27 @@ const (
 	clientConfEnvName = "SF_CLIENT_CONFIG_FILE"
 )
 
-func getClientConfig(filePathFromConnectionString string) (*ClientConfig, error) {
-	configPredefinedFilePaths := clientConfigPredefinedDirs()
-	filePath := findClientConfigFilePath(filePathFromConnectionString, configPredefinedFilePaths)
+func getClientConfig(filePathFromConnectionString string) (*ClientConfig, string, error) {
+	defaultCfgDir := defaultConfigDirectory()
+	if defaultCfgDir == "" {
+		return nil, "", nil
+	}
+	filePath := findClientConfigFilePath(filePathFromConnectionString, []string{defaultCfgDir})
 	if filePath == "" { // we did not find a config file
-		return nil, nil
+		return nil, "", nil
 	}
-	return parseClientConfiguration(filePath)
+	config, err := parseClientConfiguration(filePath)
+	return config, filePath, err
 }
 
 func findClientConfigFilePath(filePathFromConnectionString string, configPredefinedDirs []string) string {
 	if filePathFromConnectionString != "" {
+		logger.Infof("Using Easy Logging configuration path from a connection string: %s", filePathFromConnectionString)
```

Comment:
The name easy logging is internal? The official name is `client configuration`?

Reply:
Right, changed.